svr4-tools/7-s-nail.md

```diff
@@ -0,0 +1,48 @@
+# Introduction to nail-14.9.25
+s-nail is a mail user agent derived from Berkeley Mail 8.1.
+
+## Package Information
+- Download (HTTP): https://ftp.sdaoden.eu/s-nail-14.9.25.tar.xz
+- Download md5sum: 59d5818c54246d22f8ff599b8783250d
+- Download Size: 788K
+- Build Size: 7.7M
+- Build Time: 0.2 SBU
+
+## Package Dependencies
+### Required
+  [An MTA](https://www.linuxfromscratch.org/blfs/view/svn/server/mail.html) (at runtime)
+
+### Optional
+  [MIT Kerberos](https://www.linuxfromscratch.org/blfs/view/svn/postlfs/mitkrb.html)
```

Comment:
Don't forget to add an `and` after MIT Kerberos as another dependency comes after it.